bubble_sort.rb

```diff
@@ -0,0 +1,20 @@
+def bubble_sort(arr)
+  a = arr.length
+  while a.positive?
+    no_swaps = true
+    b = 0
+    while b < a - 1
+      if arr[b] > arr[b + 1]
+        arr[b], arr[b + 1] = arr[b + 1], arr[b]
+        no_swaps = false
+      end
+      b += 1
+    end
+
+    break if no_swaps
+
+    a -= 1
+  end
+  arr
+end
+puts bubble_sort([100, 243, 210, 120, 40, -80, -300, 2, 4])
```

Comment:
- [ ] Please delete all debugging code (all puts, prints, etc are debugging code if they serve to "test" your code). 
![image](https://user-images.githubusercontent.com/53256139/91252472-47a3a500-e723-11ea-99eb-ff7623b67d71.png)